ga4gh/datamodel/reads.py

```diff
@@ -187,6 +187,13 @@ def toProtocolElement(self):
         stats.baseCount = None  # TODO requires iterating through all reads
         readGroup.stats = stats
         readGroup.programs = self.getPrograms()
+        experiment = protocol.Experiment()
+        experiment.id = 'TODO'
```

Comment:
where are these values supposed to come from?  the sam header does not have most of them... metadata, presumably?